customtransforms/custom_single_json_retriever.py

```diff
@@ -0,0 +1,59 @@
+from transformer import Transformer
+from utilities import flat
+
+
+class SingleJsonRetriever(Transformer):
+    '''
+    Transforms perfherder data into the standardized data format.
+    '''
+    entry_number = 0
+
+    def transform(self, data):
```

Comment:
After making the modifications mentioned below, you should add a comment to explain how this transform works since it's a bit more complicated than anything we've seen so far.